website/versioned_docs/version-0.2.x/client-setup.mdx

```diff
@@ -16,9 +16,7 @@ In order to interact with a composite at runtime, it is first necessary to creat
   ]}>
   <TabItem value="api">
 
-Make sure you have the following packages installed, before running this code:
- * `@ceramicnetwork/http-client`
- * `@composedb/devtools-node`
+Make sure you have the `composedb` packages [installed](/installation.mdx), before running the code below.
```

Comment:
Don't make absolute links to other pages please, they should be relative so they work across versions